### PR TITLE
blog: tilt ci blog post

### PR DESCRIPTION
**Pull request by nicks**
Hello @maiamcc, @ellenkorbes, @victorwuky,

Please review the following commits I made in branch nicks/ch6051/ci2:

d7271a6eb8c26657bb9552d3c7614417b9d0db41 (2020-04-15 20:35:47 -0400)
blog: tilt ci blog post

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: blog/_posts/2020-04-16-how-to-not-break-server-startup.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+For specific details on how to get a Kind cluster set up on your own CI environment, see:
+
+- [Kind on CircleCI with Tilt](https://github.com/windmilleng/kind-local/blob/master/.circleci/README.md)
+- [Kind with Tilt](https://github.com/windmilleng/kind-local/blob/master/README.md)
+- [Other Kind on CI Examples](https://github.com/kind-ci/examples)
+
+## See it in Action
+
+`tilt ci` is available in Tilt v0.13.0 and higher.
```
Comment (victorwuky) [inline blog/_posts/2020-04-16-how-to-not-break-server-startup.md:104]:
```suggestion
`tilt ci` is available in [Tilt v0.13.0](https://github.com/windmilleng/tilt/releases/tag/v0.13.0) and higher.
```

**Comment by victorwuky** [comment]
@nicks: I suggest compressing images for static content. Every bit helps. Here’s a compressed version of the header. 
header jpg

File: blog/_posts/2020-04-16-how-to-not-break-server-startup.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+You commit a change. You push it to CI. All tests pass. You merge. The change
+deploys to staging.
+
+Then your Ops team notifies you that all the staging servers are down. But don't
+worry, they've already reverted your change!
+
+😭😭😭
+
+Oh, the war stories I could tell you about staging servers that failed to
+start. I remember one bug where the code assumed process IDs were psuedo-random,
```
Comment (ellenkorbes) [inline blog/_posts/2020-04-16-how-to-not-break-server-startup.md:27]: Typo

Comment (nicks) [reply blog/_posts/2020-04-16-how-to-not-break-server-startup.md]: fixed!

**Comment by ellenkorbes** [comment]
LGTM, but IMHO it lacks a bit of human touch at the end. We could use a sentence or two along the lines of "I hope this helps you with [problem], so that you can achieve a better [good thing]. And don't hesitate to reach out via [communication method] if you have any [type of issue expected with this]!"

**Pull request merged**